test/files/pos/i6705.scala

```diff
@@ -0,0 +1,15 @@
+trait StringTempl {
+  def mkString: String
+  def mkString(x: String): String
+}
+
+object Test {
+  implicit class extension(val x: String) {
+    def shouldBe(y: String): Boolean = ???
+  }
+
+  def test(tmpl: StringTempl): Unit = {
+    tmpl.mkString shouldBe "hello"                       // error
```

Comment:
is it a dumb question to ask why it says error? should the test framework verify `// error` comments for any "test kind"?

Reply:
It has been commented that keeping error annotations shows what was fixed. (I use `was:` comments to document previous behavior.)